shipments_tracker/app/views/shipments/show.html.erb

```diff
@@ -1,9 +1,78 @@
-<div>
+<div class="h-20 flex items-center ml-14 text-sky-700 text-sm font-semibold">
+  <a><%= link_to 'Mis envíos', shipment_path(@shipment) %></a>
```

Comment:
If you're using a `link_to`, it's unnecessary to use another `<a>` tag, as the browser will display both `anchor` labels accordingly.

Reply:
done